src/app/src/util/constants.js

```diff
@@ -434,6 +434,10 @@ export const claimAFacilityFormFields = Object.freeze({
         id: 'company-name',
         label: 'Official name of LLC or company registered',
     }),
+    parentCompany: Object.freeze({
+        id: 'parent-company',
+        label: 'Parent company of facility, selected from contributors list',
```

Comment:
Consider simplifying this label to "Parent company" and adding an `aside`. This was a bit of early feedback when reviewing the feature on a previous checkin call. Something like:

>If you cannot find the parent company in this list consider inviting them to register with the Open Apparel Registry.

<img width="1392" alt="Screen Shot 2019-06-24 at 1 58 47 PM" src="https://user-images.githubusercontent.com/17363/60051761-90c41900-9688-11e9-960b-a70f8b26021e.png">

Reply:
That makes sense. I adjusted this on both the initial form and the update page in 364ce9d79f00f30b38865c63e3f3f3fc78df982d


![Screen Shot 2019-06-24 at 5 39 11 PM](https://user-images.githubusercontent.com/4165523/60054439-8c0f5d00-96a8-11e9-8e0f-9c144d273884.png)
![Screen Shot 2019-06-24 at 5 47 36 PM](https://user-images.githubusercontent.com/4165523/60054440-8c0f5d00-96a8-11e9-87ef-33123d8b0c80.png)